File: spark_project/src/main/scala/utils/DateUtils.scala
package utils

import java.util.Date
import org.apache.commons.lang3.time.FastDateFormat
object DateUtils {
  val OLD_FORMAT=FastDateFormat.getInstance("yyyy-MM-dd HH:mm:ss")
  val TARGET_FORMAT=FastDateFormat.getInstance("yyyyMMddHHmmss")

  def getTime(time:String) ={
    OLD_FORMAT.parse(time).getTime()
  }

  def parseTime(time:String) = {
    TARGET_FORMAT.format(new Date(getTime(time)))
  }
}

File: spark_web/src/main/java/com/luochan/spark_web/utils/DateUtils.java
package com.luochan.spark_web.utils;

import java.text.ParseException;
import java.text.SimpleDateFormat;
import java.util.Date;
public class DateUtils {
    public static Date getCurrenttime()
    {
        SimpleDateFormat df = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");//设置日期格式
        String date = df.format(new Date());
        try {
            Date getNewsTime=df.parse(date);
            return getNewsTime;
        } catch (ParseException e) {
            e.printStackTrace();
        }
        return null;
    }
    public static String getCurrentTimeStr()
    {
        // getInRecTimestamp(1);
        SimpleDateFormat df = new SimpleDateFormat("yyyyMMddHHmmss");//设置日期格式
        return df.format(getCurrenttime());
    }
}
